### PR TITLE
Feature: Albanian translation

### PR DESCRIPTION
**Pull request by Aizen93**
Hello,

### Contains

this PR contains the Albanian translation of the game.

- menu_sq.lang : was added to the engine ressources assets folder 

- flag_sq.png : ALbanian flag added to the assets flags folder

### How to test

- Open the game

- Go to the Settings

- Choose Albanian in the langages menu list

- play the game to see translation

Thank you.

Aizen93 and fortiseven

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

**Comment by msteiger** [comment]
test this please

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: engine/src/main/resources/assets/i18n/menu_sq.lang [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+    "behavior-editor-paste": "ngjit",
+    "behavior-editor-remove": "heq",
+    "binding-attack": "Sulmoj",
+    "binding-autoMove-mode": "Kunj Veprimin Automatik",
+    "binding-backwards": "Mbrapsht",
+    "binding-behavior-editor": "Toggle Behavior Editor",
+    "binding-chat": "Kunj Bisedës",
+    "binding-console": "Kunj Konsolës",
+    "binding-crouch": "Struku / Ulu",
+    "binding-crouch-mode": "Në Mënyrë E Strukur,
```
Comment (msteiger) [inline engine/src/main/resources/assets/i18n/menu_sq.lang]:
```suggestion
    "binding-crouch-mode": "Në Mënyrë E Strukur",
```

File: engine/src/main/resources/assets/i18n/menu_sq.lang [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+    "telemetry-menu": "Meny e Metrikavë",
+    "telemetry-description": "Ky ekran tregon metrikat dhe vlerat e tyre që do të dërgohen në server. Ju mund ta aktivizoni / çaktivizoni këtu. Telemetri është super e dobishme për nje projekt i çilur per të gjith si i yni dhe ne do ta vlerësonim me të vërtetë!",
+    "telemetry-block-destroyed": "Blloku u shkatërrua",
+    "telemetry-block-placed": "Blloku u vendos",
+    "telemetry-game-configuration": "Rrëgullimet e Lojrës",
+    "telemetry-game-play": "Statistikat e Lojrës",
+    "telemetry-creature-killed": "Kafsha u vra",
+    "telemetry-system-context": "Konteksti i Sistemit",
+    "telemetry-modules": "Modulet",
+    "this-language-English": "Shqip",
```
Comment (msteiger) [inline engine/src/main/resources/assets/i18n/menu_sq.lang]:
```suggestion
    "this-language-English": "Albanian",
```

**Comment by msteiger** [comment]
I've added two small fixes that should make it work properly.

**Comment by Aizen93** [comment]
I just commited the fixes thanks for marking that one. but Albanian doesnt show on the langage menu list at least on my computer, i'll double check again.

**Comment by Aizen93** [comment]
Okk all tests good, everything is Ok, only thing left is some one to accept the pull request :)
We will be adding two more other langages (Updating french because its incomplete and Arabic too)

**Comment by msteiger** [comment]
Thanks, Arabic isn't displayed currently, because the text rendering engine does not yet support it. But an update on French would be good, I assume.

**Comment by Cervator** [comment]
Hi @Aizen93 and thanks! :-)

Yep I think more translations would be fine, even if they're "parked" like Arabic until #2677 is resolved 👍

**Comment by Aizen93** [comment]
@Cervator i'll make a pull request soon for the french langage
@msteiger thank you for the fixes and the merge :)

**Pull request merged**